zipkin-ui/js/component_ui/traceToMustache.js

```diff
@@ -106,8 +106,12 @@ export default function traceToMustache(trace) {
       const spanStartTs = span.timestamp || traceTimestamp;
       const spanDepth = spanDepths[span.id] || 1;
       const width = (span.duration || 0) / summary.duration * 100;
+      let isError = false;
```

Comment:
@eirslett but a.key is inside of a closure, I don't think you will be able to reassign the const each time.